package.json

```diff
@@ -50,13 +50,17 @@
     "start:server": "webpack-dev-server --open --mode development --config config/webpack.dev.mjs",
     "watch:templates": "gulp watch",
     "start": "npm run build:templates && run-p start:server watch:templates",
-    "test": "karma start karma.config.js",
+    "lint": "eslint .",
+    "test:unit": "karma start karma.config.js",
     "build": "npm run build:templates && webpack --mode production --config config/webpack.prod.mjs",
     "start:prod-local": "cd dist && http-server -p 4200 -c-1 -b",
     "prettier": "prettier --write \"**/*.{js,json,css,scss,less,md,ts,html}\"",
-    "lint": "eslint .",
+    "test:e2e": "playwright test",
+    "test:e2e:debug": "playwright test --project chromium --headed",
+    "test:e2e:report": "npx playwright show-report",
+    "test": "run-s test:e2e lint test:unit",
```

Comment:
The unit test must be run before the e2e test, because if you run the unit tests before e2e (single run without watch mode), as soon as the unit tests pass, the script will exit without running the e2e tests.

<img width="823" alt="image" src="https://user-images.githubusercontent.com/5073455/224019847-6750b273-70f4-475b-805f-37371196a662.png">

The problem does not arise if we run e2e before unit tests. Then both processes will execute

<img width="952" alt="image" src="https://user-images.githubusercontent.com/5073455/224020409-9c2686f3-2534-470e-b823-492e78d32997.png">

Reply:
Done